test/node_functions.js

```diff
@@ -20,6 +20,15 @@ var puts = console.log;
 puts("Running tests...");
 
 var tests = {
+  // R6RS core
+  "let ()": function(){
+    assert.ok(ev('(let () #t)'));
+  },
+
+  "let* ()": function(){
+    assert.ok(ev('(let* () #t)'));
+  },
+
```

Comment:
Actually this file is for Node.js-related tests.
Could you move them to test/unit.js (say, [here](https://github.com/biwascheme/biwascheme/blob/master/test/unit.js#L380-L389))?

Reply:
Somehow I completely missed test/unit.js! I've moved them there, now.